activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   When using `has_many_through`, building records on the association, then deleting them
+    and then saving the parent record will no longer save the previously deleted records.
+
+    *Tobias Kraze*
```

Comment:
`has_many_through` should be `has_many :through`. I also think the sentence is a little bit of a run on and would read clearer as:

> Previously when building records using a `has_many :through` association, if the child records were deleted before the parent was saved they would be persisted. Now if child records are deleted before the parent is saved on a `has_many :through` association the child records will now be deleted.